api/envoy/config/route/v3/route_components.proto

```diff
@@ -1785,6 +1785,25 @@ message RateLimit {
           "envoy.api.v2.route.RateLimit.Action.RemoteAddress";
     }
 
+    // The following descriptor entry is appended to the descriptor and is populated using the
+    // masked address from :ref:`x-forwarded-for <config_http_conn_man_headers_x-forwarded-for>`:
+    //
+    // .. code-block:: cpp
+    //
+    //   ("masked_remote_address", "<masked address from x-forwarded-for>")
+    message MaskedRemoteAddress {
```

Comment:
This will need a release note.

Reply:
add release-notes need rebase of main branch due to https://github.com/envoyproxy/envoy/pull/21075 changes, I'll do that after all code are reviewed.

Reply:
The doc seems to imply that header `x-forwarded-for` is used but the implementation just uses connection info. Is that expected?